Moodlet/app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#008577</color>
     <color name="colorPrimaryDark">#00574B</color>
     <color name="colorAccent">#D81B60</color>
+    <color name="colorLoginBlue">#2699FB</color>
```

Comment:
i think this color is gonna be used often so maybe set it to the accent? Instead of specifically saying "colorLoginBlue"

Reply:
done